docs/WritingTests.asciidoc

```diff
@@ -191,25 +191,29 @@ clone_job.pl --from localhost 42 _GROUP=0
 === Using snapshots to speed up development of tests
 [id="snapshots"]
 
-Sometimes it's annoying to run the full installation to adjust some
-test. It would be nice to have the VM jump to a certain point. +QEMU
-backend+ provides feature that allows a job to start from a snapshot that might
-help in this situation.
+Sometimes it is annoying to run the full installation to adjust some
+test. It would be nice to have the VM jump to a certain point. The +QEMU
+backend+ provides a feature that allows a job to start from a snapshot which
+can help in this situation.
```

Comment:
I don't really like this paragraph. i don't know how to improve it, but it feels too much informal.